_includes/docs/latest/setup-cloudsql.md

```diff
@@ -26,10 +28,26 @@ To set up your Cloud SQL instance for Forseti, follow the steps below:
     to proxy your connection to your Cloud SQL instance. Your
     INSTANCE_CONNECTION_NAME is the **Instance Connection Name** under
     **Properties** on the Cloud SQL dashboard instance details, in the format "PROJECTID:REGION:INSTANCEID".
+    
+    If you are using Cloud SDK authentication:
 
       ```bash
       <path/to/cloud_sql_proxy> -instances=INSTANCE_CONNECTION_NAME=tcp:3306
       ```
+    
+    If you are using a service account to authenticate (recommended for production environments):
+    
+      ```bash
+      <path/to/cloud_sql_proxy> -instances=<INSTANCE_CONNECTION_NAME>=tcp:3306 \
+                  -credential_file=<PATH_TO_KEY_FILE>
+      ```
+    
+    For example:
+    
+      ```bash
+      ./cloud_sql_proxy -instances=ebeach-forseti-dev-01:us-central1:mysql-instance=tcp:3306 \
```

Comment:
We should not be using our actual resource names.  We typically do using something fake, like `foo` or `foobar`.  So, how about

```
foobar-forseti-dev-01:us-central1:mysql-instance
```
```
foobar/foobar-forseti-dev-01-0d152c0e1bc8.json
```